A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -707,7 +707,7 @@ class FMCMainDisplay extends BaseAirliners {
 
         let altToUse = this.cruiseFlightLevel;
         // Use the cruise level for calculations otherwise after cruise use descent altitude down to 10,000 feet.
-        if ([FlightPhase.FLIGHT_PHASE_DESCENT, FlightPhase.FLIGHT_PHASE_APPROACH, FlightPhase.FLIGHT_PHASE_GOAROUND].indexOf(this.currentFlightPhase) + 1) {
+        if ([FlightPhase.FLIGHT_PHASE_DESCENT, FlightPhase.FLIGHT_PHASE_APPROACH, FlightPhase.FLIGHT_PHASE_GOAROUND].includes(this.currentFlightPhase)) {
```

Comment:
`this.currentFlightPhase > FlightPhases.FLIGHT_PHASE_CRUISE`

Reply:
Thanks, still learning Enums lol. Fixed in 704dc35